twitter-clone/app/src/main/java/io/github/vnicius/twitterclone/data/datasource/usertweets/UserTweetsDataSource.kt

```diff
@@ -0,0 +1,67 @@
+package io.github.vnicius.twitterclone.data.datasource.usertweets
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.paging.PageKeyedDataSource
+import io.github.vnicius.twitterclone.data.repository.user.UserRepository
+import io.github.vnicius.twitterclone.utils.State
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.launch
+import twitter4j.Status
+import java.lang.Exception
+
+class UserTweetsDataSource(
+    private val userId: Long,
+    private val itemsCount: Int,
```

Comment:
If the `itemsCount` means the page size I think it makes more sense to use the name `pageSize`.